doc/python/mapbox-layers.md

```diff
@@ -43,6 +43,7 @@ If your figure contains one or more traces of type `go.Scattermapbox`, `go.Choro
   2. The various traces in `data` are by default rendered above the base map (although this can be controlled via the `below` attribute).
   3. `layout.mapbox.layers` is an array that defines more layers that are by default rendered above the traces in `data` (although this can also be controlled via the `below` attribute).
 
+In respect to [attribution instructions](https://docs.mapbox.com/help/how-mapbox-works/attribution/), we added [sourceattribution](https://plot.ly/python/reference/#layout-mapbox-layers-items-layer-sourceattribution) attribute, which displays both `mapbox wordmark` and `text attribution` on our maps.
```

Comment:
It would be great if we could include an example chart for this attribute. I would also suggest rephrasing this to:

In order to  [add attribution information to charts](https://docs.mapbox.com/help/how-mapbox-works/attribution/), please use the [sourceattribution](https://plot.ly/python/reference/#layout-mapbox-layers-items-layer-sourceattribution) attribute, which allows you to display both `mapbox wordmark` and `text attribution` on maps.

Reply:
+1